File: Python3/프로그래머스/0/181890. 왼쪽 오른쪽/왼쪽 오른쪽.py
def solution(str_list):
    answer = []
    
    for index in range(len(str_list)):
        if str_list[index] == "l":
            answer = str_list[0:index]
            break
        elif str_list[index] == "r":
            answer = str_list[index+1:]
            break
            
    return answer

File: Swift/프로그래머스/0/181890. 왼쪽 오른쪽/왼쪽 오른쪽.swift
import Foundation

func solution(_ str_list:[String]) -> [String] {
    var answer: [String] = []
    
    for index in 0..<str_list.count {
        if str_list[index] == "l" {
            answer = Array(str_list[0..<index])
            break
        } else if str_list[index] == "r" {
            answer = Array(str_list[(index+1)..<str_list.count])
            break
        }
    }
    
    return answer
}